framework/utils/randomUtils.py

```diff
@@ -200,20 +201,36 @@ def randomIntegers(low, high, caller=None, engine=None):
   else:
     raise TypeError('Engine type not recognized! {}'.format(type(engine)))
 
-def randomChoice(array, engine=None):
+def randomChoice(array, size = 1, replace = True, engine = None):
```

Comment:
It would be better if randomChoice had a unit test.

Reply:
unit test has been created